### PR TITLE
Updated new prereq for CSM Operator offline bundle creation

### PR DESCRIPTION
**Pull request by rajendraindukuri**
# Description
Updated new prereq for CSM Operator offline bundle creation

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1435|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?

### CODE REVIEW

File: content/docs/deployment/csmoperator/_index.md [hunk 1]
```diff
@@ -152,6 +153,7 @@ To perform an offline installation, the following steps should be performed:
 >NOTE: It is recommended to use the same build tool for packing and unpacking of images (either docker or podman).
 
 #### Building an offline bundle
+>NOTE: Please login to `registry.redhat.io` registry using Redhat credentials before you proceed with offline bundle creation.
```
Comment (donatwork) [inline content/docs/deployment/csmoperator/_index.md:156]: There is this tendency to be over polite in documenting steps. You are providing steps for a procedure not having a conversation. You do not need to start sentences with please. Just state "Login to ..." and that is more concise.

Comment (rajendraindukuri) [reply content/docs/deployment/csmoperator/_index.md]: Done

File: content/docs/deployment/csmoperator/_index.md [hunk 2]
```diff
@@ -152,6 +153,7 @@ To perform an offline installation, the following steps should be performed:
 >NOTE: It is recommended to use the same build tool for packing and unpacking of images (either docker or podman).
 
 #### Building an offline bundle
+>NOTE: Login to `registry.redhat.io` registry using Redhat credentials before you proceed with offline bundle creation.
```
Comment (donatwork) [inline content/docs/deployment/csmoperator/_index.md:156]: RedHat, uppercase H. Also "Login to **the**..."

File: content/docs/deployment/csmoperator/_index.md [hunk 3]
```diff
@@ -126,6 +126,7 @@ The `csm-offline-bundle.sh` script can be used to create a package usable for of
 Multiple Linux-based systems may be required to create and process an offline bundle for use.
 * One Linux-based system, with Internet access, will be used to create the bundle. This involves the user cloning a git repository hosted on github.com and then invoking a script that utilizes `docker` or `podman` to pull and save container images to file.
 * One Linux-based system, with access to an image registry, to invoke a script that uses `docker` or `podman` to restore container images from file and push them to a registry
+* Redhat credentials to pull `openshift4/ose-kube-rbac-proxy-rhel9` image from `registry.redhat.io` (This registry does not support unauthenticated access)
```
Comment (donatwork) [inline content/docs/deployment/csmoperator/_index.md:129]: RedHat

**Review by bpjain2004 — COMMENTED** [review]
Don't we need to add similar instructions in the following section as well?
https://dell.github.io/csm-docs/docs/deployment/offline/drivers/#building-an-offline-bundle

**Review by AronAtDell — APPROVED** [review]
Apart from the comments from Don. Everything looks fine.

File: content/docs/deployment/csmoperator/_index.md (hunk 2 above)
Comment (tdawe) [reply content/docs/deployment/csmoperator/_index.md:156]: Fixed

**Comment by tdawe** [comment]
> Don't we need to add similar instructions in the following section as well? https://dell.github.io/csm-docs/docs/deployment/offline/drivers/#building-an-offline-bundle

Added the same docs to that page.

**Review by sharmilarama — APPROVED** [review]

**Pull request merged**